src/uwtools/cli.py

```diff
@@ -305,6 +306,121 @@ def _dispatch_forecast_run(args: Namespace) -> bool:
     )
 
 
+# Mode Rocoto
+
+
+def _add_subparser_rocoto(subparsers: Subparsers) -> ModeChecks:
+    """
+    Subparser for mode: rocoto
+
+    :param subparsers: Parent parser's subparsers, to add this subparser to.
+    """
+    parser = _add_subparser(subparsers, STR.rocoto, "Write and validate Rocoto XML Documents")
+    _basic_setup(parser)
+    subparsers = _add_subparsers(parser, STR.submode)
+    return {
+        STR.write: _add_subparser_rocoto_write(subparsers),
+        STR.validate: _add_subparser_rocoto_validate(subparsers),
+    }
+
+
+def _add_subparser_rocoto_write(subparsers: Subparsers) -> SubmodeChecks:
+    """
+    Subparser for mode: rocoto write
+
+    :param subparsers: Parent parser's subparsers, to add this subparser to.
+    """
+    parser = _add_subparser(subparsers, STR.write, "Write a Rocoto XML workflow document")
+    required = parser.add_argument_group(TITLE_REQ_ARG)
+    _add_arg_input_file(required)
+    _add_arg_schema_file(required)
```

Comment:
I don't think the user should have to provide this schema file. A rocoto YAML should always be validated against our  Rocoto JSONSchema file.

Reply:
At one point I had this as optional too, but that's fair